lib/secure_headers.rb

```diff
@@ -163,7 +163,7 @@ def opt_out_of_all_protection(request)
     # returned is meant to be merged into the header value from `@app.call(env)`
     # in Rack middleware.
     def header_hash_for(request)
-      config = config_for(request, prevent_dup = true)
+      config = config_for(request, _prevent_dup = true)
```

Comment:
Strange. It's [definitely used](https://github.com/twitter/secureheaders/blob/fd559f2acc591c04e0c9563e3f9e857e7ce6e6c9/lib/secure_headers.rb#L233) and important.

As a side note: I'm ok with making strange changes to appease a tool in the name of keeping a clean list of warnings.